source/docs/software/advanced-controls/filters/debouncer.rst

```diff
@@ -1,7 +1,21 @@
 Debouncer
 =========
 
-The ``Debouncer`` class is used to filter out unwanted quick on/off cycles.  These cycles are usually due to a sensor error like noise or reflections and not the actual event the sensor is trying to record.  The ``Debouncer`` works by only recording a change in signal after that change has persisted for a specified period of time.
+A debouncer is a filter used to eliminate unwanted quick on/off cycles (termed "bounces"). These cycles are usually due to a sensor error like noise or reflections and not the actual event the sensor is trying to record.
```

Comment:
The term bounce refers to physical limit switches bouncing against the contacts when they are closed.

Reply:
Originally, yeah; the actual effect of the bouncing switch is the cycles that are described here